### ISSUE TITLE
[REVIEW]: LazySets.jl: Scalable Symbolic-Numeric Set Computations

### ISSUE THREAD

**Issue by whedon** [JuliaCon/proceedings-review#97]
**Submitting author:** @mforets (<a href="http://orcid.org/0000-0002-9831-7801">Marcelo Forets</a>)
**Repository:** <a href="https://github.com/JuliaReach/LazySets-JuliaCon21" target ="_blank">https://github.com/JuliaReach/LazySets-JuliaCon21</a>
**Version:** v1.53.4
**Reviewer:** @gdalle, @blegat
**Archive:** <a href="https://doi.org/10.5281/zenodo.5761068" target="_blank">10.5281/zenodo.5761068</a>

## Status

[![status](https://proceedings.juliacon.org/papers/681437dbee39acec76df25669b3a3542/status.svg)](https://proceedings.juliacon.org/papers/681437dbee39acec76df25669b3a3542)

Status badge code:

```
HTML: <a href="https://proceedings.juliacon.org/papers/681437dbee39acec76df25669b3a3542"><img src="https://proceedings.juliacon.org/papers/681437dbee39acec76df25669b3a3542/status.svg"></a>
Markdown: [![status](https://proceedings.juliacon.org/papers/681437dbee39acec76df25669b3a3542/status.svg)](https://proceedings.juliacon.org/papers/681437dbee39acec76df25669b3a3542)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/JuliaReach/LazySets-JuliaCon21" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@gdalle & @blegat, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @matbesancon know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [ ] As the reviewer I confirm that I have read the [JuliaCon conflict of interest policy](https://proceedings.juliacon.org/guide/authors#coi) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [ ] I confirm that I read and will adhere to the [JuliaCon code of conduct](https://proceedings.juliacon.org/coc).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JuliaReach/LazySets-JuliaCon21">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Authorship:** Has the submitting author (@mforets) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Paper format

- [ ] **Authors:** Does the `paper.tex` file include a list of authors with their affiliations?
- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?
- [ ] **Page limit:** Is the page limit for full papers respected by the submitted document?

### Content

- [ ] **Context:** is the scientific context motivating the work correctly presented?
- [ ] **Methodology:** is the approach taken in the work justified, presented with enough details and reference to reproduce it?
- [ ] **Results:** are the results presented and compared to approaches with similar goals?
### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JuliaCon conflict of interest policy](https://proceedings.juliacon.org/guide/authors#coi) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JuliaCon code of conduct](https://proceedings.juliacon.org/coc).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JuliaReach/LazySets-JuliaCon21">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Authorship:** Has the submitting author (@mforets) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Paper format

- [x] **Authors:** Does the `paper.tex` file include a list of authors with their affiliations?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?
- [x] **Page limit:** Is the page limit for full papers respected by the submitted document?

### Content

- [x] **Context:** is the scientific context motivating the work correctly presented?
- [x] **Methodology:** is the approach taken in the work justified, presented with enough details and reference to reproduce it?
- [x] **Results:** are the results presented and compared to approaches with similar goals?

**Comment by matbesancon** [comment]
when you do some modifications, you can rerun compilation yourself with `@whedon generate pdf`

**Comment by mforets** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by mforets** [comment]
> the section on a statement of need. This is fine if this doesn't have this name but there should be at least a paragraph in the paper stating it

The statement of need can be found in paragraph 2 of the Introduction (Section 1). We just made a small change describing who the target audience is.

**Comment by mforets** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.29007/zkf6 is OK
- 10.29007/7dt2 is OK
- 10.5281/zenodo.4993670 is OK
- 10.1137/15M1020575 is OK
- 10.1145/3447928.3456704 is OK
- 10.1007/978-3-030-89716-1_6 is OK
- 10.1109/MEMOCODE51338.2020.9314994 is OK
- 10.1137/141000671 is OK
- 10.1146/annurev-control-071420-081941 is OK
- 10.1145/3178126.3178128 is OK
- 10.1007/978-3-319-57288-8_20 is OK
- 10.5281/zenodo.2613102 is OK
- 10.29007/zbkv is OK
- 10.1007/978-3-642-22110-1_30 is OK
- 10.1109/CDC.2006.377036 is OK
- 10.1145/3302504.3311804 is OK
- 10.1561/2400000035 is OK
- 10.1109/TCAD.2020.3012859 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by blegat** [comment]
The paper is well written, I would recommend acceptance with minor revision. My comments are the following:

* Page 1 line 13 of right column: "comment" -> "comment**s**"
* "Optimization over a polyhedron corresponds to solving a linear program" -> this assumes that the objective function is linear
* "parameters to represent" should probably be "parameters that represent" or "parameters needed to represent"
* “common functionality” -> “common functionalities” or “a common functionality”
* “fresh set” -> “new set” or “fresh new set”
* In the LaTeX formula at the beginning of the last paragraph of page 3, none of the symbols are defined, I would say that CH is defined in Table 1, X0, E_+ are sets and Phi is a matrix (as defined later).
* In the definition of the support function, “maximum” should be supremum unless necessary assumptions are met for X
* “over polygon” -> “over the polygon” or “over a polygon”
* “numeric queries”: what do you mean ? Do you mean “numerical queries” ?
* Section 4.2, “Fig. 2 (left)” should be “Fig. 2 (right)”
* In Section 5.1, the first equation does not render well. It should be either in LaTeX math or in monospace font (e.g. wth texttt)
* In caption of Fig. 5, the first “(right)” should be “(left)”
* In the second line of the right column of page 10, “d1” and “d2” are not define. Moreover, d should have length 2n in this equation while it is defined to have length n
* The reference to MathOptInterface should be update: it is now published and contains a DOI (since a few days ago). “MathOptInterface” should be capitalized
* In the reference of TaylorModels.jl, “Taylormodels.jl” -> M should be capitalized

**Comment by schillic** [comment]
@blegat Thanks for the thorough reading! We added https://github.com/JuliaReach/LazySets-JuliaCon21/pull/2 to address the review. Feel free to comment there.

**Comment by schillic** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by gdalle** [comment]
## General review

This paper introduces a very rich and thoughtfully-crafted package for set representation. The key features of LazySets.jl are clearly stated and illustrated with very visual examples. Its natural syntax, high performance, integration into the Julia ecosystem and extensive documentation are tremendous assets. **I recommend acceptance with minor revision.**

To make the paper even better, the authors could emphasize why the core functionalities of their package (like support function computations) are precisely the ones we need, especially in fields like convex analysis or reachability. A more thorough comparison with competing libraries would also be welcome. Provided these two points are addressed, I will be able to check the remaining boxes in the Context section below.

For some reason I am not able to edit the checklist, so I reproduce it below.

## Reviewer checklist

### Conflict of interest

- [x] As the reviewer I confirm that I have read the JuliaCon conflict of interest policy and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the JuliaCon code of conduct.

### General checks

- [ ] Repository: Is the source code for this software available at the [repository url](https://github.com/JuliaReach/LazySets-JuliaCon21)?
- [x] License: Does the repository contain a plain-text LICENSE file with the contents of an OSI approved software license?
- [x] Authorship: Has the submitting author (@mforets) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] Installation: Does installation proceed as outlined in the documentation?
- [x] Functionality: Have the functional claims of the software been confirmed?
- [x] Performance: If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] A statement of need: Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] Installation instructions: Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] Example usage: Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] Functionality documentation: Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] Automated tests: Are there automated tests or manual steps described so that the function of the software can be verified?
- [x] Community guidelines: Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Paper format

- [x] Authors: Does the paper.tex file include a list of authors with their affiliations?
- [x] A statement of need: Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] References: Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?
- [x] Page limit: Is the page limit for full papers respected by the submitted document?

### Content

- [ ] Context: is the scientific context motivating the work correctly presented?
- [x] Methodology: is the approach taken in the work justified, presented with enough details and reference to reproduce it?
- [ ] Results: are the results presented and compared to approaches with similar goals?

**Comment by gdalle** [comment]
I have put more detailed comments in this issue https://github.com/JuliaReach/LazySets-JuliaCon21/issues/4

**Comment by whedon** [comment]
:wave: @blegat, please update us on how your review is going (this is an automated reminder).

**Comment by whedon** [comment]
:wave: @gdalle, please update us on how your review is going (this is an automated reminder).

**Comment by gdalle** [comment]
I am waiting on a revised version by @mforets and @schillic to check the remaining boxes

**Comment by schillic** [comment]
@gdalle: We have started with the revision but are not finished yet. We will let you know.

**Comment by mforets** [comment]
@gdalle: we have replied in https://github.com/JuliaReach/LazySets-JuliaCon21/pull/5

**Comment by gdalle** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by schillic** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by schillic** [comment]
@gdalle: We merged the PR so you can see the new version here.

**Comment by gdalle** [comment]
Lets try this again then

**Comment by gdalle** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by matbesancon** [comment]
@mforets let us know when everything is addressed from the reviews on your side

**Comment by gdalle** [comment]
My second review pass is available here: https://github.com/JuliaReach/LazySets-JuliaCon21/pull/5#issuecomment-986201407

The changes made by the authors significantly improve the clarity of the paper. I still have a handful of very minor formatting issues (see issue link) but I can now recommend **acceptance**

**Comment by mforets** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by schillic** [comment]
@matbesancon I believe we fixed all the items raised in the reviews. (Recall that @gdalle was not able to modify the OP checklist.)
What is the next step?

**Comment by matbesancon** [comment]
@whedon check references

**Comment by matbesancon** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List all of Whedon's capabilities
@whedon commands

# Assign a GitHub user as the sole reviewer of this submission
@whedon assign @username as reviewer

# Add a GitHub user to the reviewers of this submission
@whedon add @username as reviewer

# Re-invite a reviewer (if they can't update checklists)
@whedon re-invite @username as reviewer

# Remove a GitHub user from the reviewers of this submission
@whedon remove @username as reviewer

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

# Change editorial assignment
@whedon assign @username as editor

# Set the software archive DOI at the top of the issue e.g.
@whedon set 10.0000/zenodo.00000 as archive

# Set the software version at the top of the issue e.g.
@whedon set v1.0.1 as version

# Open the review issue
@whedon start review

EDITORIAL TASKS

# All commands can be run on a non-default branch, to do this pass a custom 
# branch name by following the command with `from branch custom-branch-name`.
# For example:

# Compile the paper
@whedon generate pdf

# Compile the paper from alternative branch
@whedon generate pdf from branch custom-branch-name

# Remind an author or reviewer to return to a review after a
# certain period of time (supported units days and weeks)
@whedon remind @reviewer in 2 weeks

# Ask Whedon to do a dry run of accepting the paper and depositing with Crossref
@whedon recommend-accept

# Ask Whedon to check the references for missing DOIs
@whedon check references

# Ask Whedon to check repository statistics for the submitted software
@whedon check repository

EiC TASKS

# Invite an editor to edit a submission (sending them an email)
@whedon invite @editor as editor

# Reject a paper
@whedon reject

# Withdraw a paper
@whedon withdraw

# Ask Whedon to actually accept the paper and deposit with Crossref
@whedon accept deposit=true

```

**Comment by matbesancon** [comment]
@whedon check references

**Comment by matbesancon** [comment]
@schillic can you confirm that 10.5281/zenodo.5761068 is the DOI of the latest version (which corresponds to the paper)?

**Comment by matbesancon** [comment]
@whedon set 10.5281/zenodo.5761068 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.5761068" target="_blank">10.5281/zenodo.5761068</a> is the archive.

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.29007/zkf6 is OK
- 10.29007/7dt2 is OK
- 10.5281/zenodo.4993670 is OK
- 10.1137/15M1020575 is OK
- 10.1287/ijoc.2021.1067 is OK
- 10.1145/3447928.3456704 is OK
- 10.1007/978-3-030-89716-1_6 is OK
- 10.1016/j.compstruc.2021.106699 is OK
- 10.1109/MEMOCODE51338.2020.9314994 is OK
- 10.1137/141000671 is OK
- 10.1146/annurev-control-071420-081941 is OK
- 10.1145/3178126.3178128 is OK
- 10.1007/978-3-319-57288-8_20 is OK
- 10.5281/zenodo.2613102 is OK
- 10.29007/zbkv is OK
- 10.1007/978-3-642-22110-1_30 is OK
- 10.1109/CDC.2006.377036 is OK
- 10.1145/3302504.3311804 is OK
- 10.1561/2400000035 is OK
- 10.1109/TCAD.2020.3012859 is OK
- 10.1515/9781400884179 is OK

MISSING DOIs

- 10.1134/s0965542508060055 may be a valid DOI for title: The modified method of refined bounds for polyhedral approximation of convex polytopes

INVALID DOIs

- None
```

**Comment by matbesancon** [comment]
@whedon set v1.53.4 as version

**Comment by whedon** [comment]
OK. v1.53.4 is the version.

**Comment by matbesancon** [comment]
Also: can you check the missing DOI and add it to the references?
https://github.com/JuliaCon/proceedings-review/issues/97#issuecomment-996584733

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.29007/zkf6 is OK
- 10.29007/7dt2 is OK
- 10.5281/zenodo.4993670 is OK
- 10.1137/15M1020575 is OK
- 10.1287/ijoc.2021.1067 is OK
- 10.1145/3447928.3456704 is OK
- 10.1007/978-3-030-89716-1_6 is OK
- 10.1016/j.compstruc.2021.106699 is OK
- 10.1109/MEMOCODE51338.2020.9314994 is OK
- 10.1137/141000671 is OK
- 10.1146/annurev-control-071420-081941 is OK
- 10.1145/3178126.3178128 is OK
- 10.1007/978-3-319-57288-8_20 is OK
- 10.5281/zenodo.2613102 is OK
- 10.29007/zbkv is OK
- 10.1007/978-3-642-22110-1_30 is OK
- 10.1109/CDC.2006.377036 is OK
- 10.1145/3302504.3311804 is OK
- 10.1561/2400000035 is OK
- 10.1109/TCAD.2020.3012859 is OK
- 10.1515/9781400884179 is OK

MISSING DOIs

- 10.1134/s0965542508060055 may be a valid DOI for title: The modified method of refined bounds for polyhedral approximation of convex polytopes

INVALID DOIs

- None
```

**Comment by schillic** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00097/jcon.00097/10.21105.jcon.00097.pdf) :page_facing_up: :point_left:

**Comment by schillic** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1134/S0965542508060055 is OK
- 10.29007/zkf6 is OK
- 10.29007/7dt2 is OK
- 10.5281/zenodo.4993670 is OK
- 10.1137/15M1020575 is OK
- 10.1287/ijoc.2021.1067 is OK
- 10.1145/3447928.3456704 is OK
- 10.1007/978-3-030-89716-1_6 is OK
- 10.1016/j.compstruc.2021.106699 is OK
- 10.1109/MEMOCODE51338.2020.9314994 is OK
- 10.1137/141000671 is OK
- 10.1146/annurev-control-071420-081941 is OK
- 10.1145/3178126.3178128 is OK
- 10.1007/978-3-319-57288-8_20 is OK
- 10.5281/zenodo.2613102 is OK
- 10.29007/zbkv is OK
- 10.1007/978-3-642-22110-1_30 is OK
- 10.1109/CDC.2006.377036 is OK
- 10.1145/3302504.3311804 is OK
- 10.1561/2400000035 is OK
- 10.1109/TCAD.2020.3012859 is OK
- 10.1515/9781400884179 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by schillic** [comment]
@matbesancon 
> can you confirm that 10.5281/zenodo.5761068 is the DOI of the latest version (which corresponds to the paper)?

confirmed

> can you check the missing DOI and add it to the references?

Yes, sorry, it's there now!

**Comment by matbesancon** [comment]
@whedon recommend-accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
:wave: @JuliaCon/jcon-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/JuliaCon/proceedings-papers/pull/49

If the paper PDF and Crossref deposit XML look good in https://github.com/JuliaCon/proceedings-papers/pull/49, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by matbesancon** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1134/S0965542508060055 is OK
- 10.29007/zkf6 is OK
- 10.29007/7dt2 is OK
- 10.5281/zenodo.4993670 is OK
- 10.1137/15M1020575 is OK
- 10.1287/ijoc.2021.1067 is OK
- 10.1145/3447928.3456704 is OK
- 10.1007/978-3-030-89716-1_6 is OK
- 10.1016/j.compstruc.2021.106699 is OK
- 10.1109/MEMOCODE51338.2020.9314994 is OK
- 10.1137/141000671 is OK
- 10.1146/annurev-control-071420-081941 is OK
- 10.1145/3178126.3178128 is OK
- 10.1007/978-3-319-57288-8_20 is OK
- 10.5281/zenodo.2613102 is OK
- 10.29007/zbkv is OK
- 10.1007/978-3-642-22110-1_30 is OK
- 10.1109/CDC.2006.377036 is OK
- 10.1145/3302504.3311804 is OK
- 10.1561/2400000035 is OK
- 10.1109/TCAD.2020.3012859 is OK
- 10.1515/9781400884179 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JCON!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/JuliaCon/proceedings-papers/pull/50
1. Wait a couple of minutes, then verify that the paper DOI resolves [https://doi.org/10.21105/jcon.00097](https://doi.org/10.21105/jcon.00097)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by matbesancon** [comment]
Thanks @gdalle and @blegat and congrats to the authors :)

**Comment by mforets** [comment]
Thank you @matbesancon for the editorial work and @gdalle and @blegat for your time reviewing our submission. 

It was our first experience with an open review process. It went so smoothly! We would highly recommend it to our colleagues.

**Comment by matbesancon** [comment]
@mforets sorry there is a rendering error: https://proceedings.juliacon.org/papers/10.21105/jcon.00097
can you remove the `$^*$` in the title and put the footnote in the text, we'll then update the paper page

**Comment by schillic** [comment]
@matbesancon we just merged https://github.com/JuliaReach/LazySets-JuliaCon21/pull/7 to remove the star. What should we do now?

**Comment by matbesancon** [comment]
Thank you, the rest will be on our side to change the paper :)

**Comment by schillic** [comment]
Thanks, and sorry for the extra work.
